python/tvm/meta_schedule/schedule_rule/multi_level_tiling.py

```diff
@@ -172,6 +174,7 @@ def __init__(
         vector_load_lens: Optional[List[int]] = None,
         reuse_read: Optional[ReuseType] = None,
         reuse_write: Optional[ReuseType] = None,
+        use_software_pipeline: bool = False,
```

Comment:
I'm not sure if we can turn on `use_software_pipeline` by default.

Reply:
it depends on the arch. It cause some regression in some cases because shared memory is small, need to use dynamic shared memory in this case (not supported yet). Will revisit this later